metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInfo.pdl

```diff
@@ -1,50 +1,46 @@
 namespace com.linkedin.dataprocess
 
-import com.linkedin.common.DatasetUrn
+import com.linkedin.common.CustomProperties
+import com.linkedin.common.ExternalReference
 
 /**
  * The inputs and outputs of this data process
  */
 @Aspect = {
   "name": "dataProcessInfo"
 }
-record DataProcessInfo {
+record DataProcessInfo includes CustomProperties, ExternalReference {
 
   /**
-   * the inputs of the data process
+   * Process name
    */
-  @Relationship = {
-    "/*": {
-      "name": "Consumes",
-      "entityTypes": [ "dataset" ]
-    }
-  }
   @Searchable = {
-    "/*": {
-      "fieldName": "inputs",
-      "fieldType": "URN",
-      "numValuesFieldName": "numInputDatasets",
-      "queryByDefault": false
-    }
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
   }
-  inputs: optional array[DatasetUrn]
+  name: string
 
   /**
-   * the outputs of the data process
+   * Process description
    */
-  @Relationship = {
-    "/*": {
-      "name": "Consumes",
-      "entityTypes": [ "dataset" ]
-    }
+  @Searchable = {
+    "fieldType": "TEXT",
+    "hasValuesFieldName": "hasDescription"
   }
+  description: optional string
+
+  /**
+   * Process type
+   */
   @Searchable = {
-    "/*": {
-      "fieldName": "outputs",
-      "fieldType": "URN",
-      "numValuesFieldName": "numOutputDatasets",
-      "queryByDefault": false
-    }
+    "fieldType": "TEXT",
+    "hasValuesFieldName": "hasType"
   }
-  outputs: optional array[DatasetUrn]
+  type: optional string
+
+  /**
+   * Status of the process
+   */
+  status: optional ProcessStatus
```

Comment:
since `status` is an operational construct, let's model it as a time-series aspect?

Reply:
makes totally sense!
addressed in https://github.com/linkedin/datahub/pull/4114/commits/aa707e6ed954301ce588553c87760597e534c764